### PR TITLE
Closes #32 #33. Add payload param to verification. Update docs.

### LINKED ISSUE #32: Add a section for transferring optional custom data.

### PR DESCRIPTION
**Pull request by artemii235**

### CODE REVIEW

**Review by hlogeon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,3 +29,7 @@ that was received from auth service.
 1. To install dependencies run: `docker-compose exec verify yarn`.
 1. To run tests run: `docker-compose exec verify yarn test`.
 1. To build production image run `docker-compose -f docker-compose.prod.yml build --no-cache`.
+
+#How to generate docs?
```
Comment (Aleserche) [inline README.md:33]: Fix it, pls. And 24 line too.

**Review by Aleserche — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (artemii235) [reply README.md:33]: @Aleserche Please clarify what to fix here?

Comment (Aleserche) [reply README.md:33]: Add a space after `#`. To be rendered correctly.

Comment (artemii235) [reply README.md:33]: @Aleserche Correct, will fix in next PR, thanks!

**Pull request merged**